cli.js

```diff
@@ -3,116 +3,162 @@
 'use strict'
 
 const fs = require('fs').promises
-const { promisify } = require('util')
+const {promisify} = require('util')
 const read = promisify(require('read'))
 const puppeteer = require('puppeteer')
 const homedir = require('os').homedir()
+const cron = require('node-cron');
 
-async function configure() {
-  console.log('parkalot-auto-reserve hasn\'t been configured yet!')
-
-  const config = {
-    username: await read({ prompt: 'Enter your username:', terminal: true }),
-    password: await read({ prompt: 'Enter your password:', silent : true, terminal: true })
-  }
-
-  await fs.writeFile(`${homedir}/.parkalotrc`, JSON.stringify(config))
-
-  return config
-}
-
-;(async () => {
-  let config
 
-  try {
-    config = JSON.parse(await fs.readFile(`${homedir}/.parkalotrc`))
-  } catch (ex) {
-    if (ex.code !== 'ENOENT') {
-      throw ex
-    }
-  }
+async function configure() {
+    console.log('parkalot-auto-reserve hasn\'t been configured yet!')
 
-  try {
-    if (!config) {
-      config = await configure()
+    const config = {
+        username: await read({prompt: 'Enter your username:', terminal: true}),
+        password: await read({prompt: 'Enter your password:', silent: true, terminal: true}),
+        spotNumber: await read({prompt: 'Enter your desired parking spot number:', silent: true, terminal: true})
     }
 
-    const browser = await puppeteer.launch({ headless: true })
-    const page = await browser.newPage()
+    await fs.writeFile(`${homedir}/.parkalotrc`, JSON.stringify(config))
 
-    await page.setViewport({
-      width: 1200,
-      height: 980,
-      deviceScaleFactor: 1,
-    })
+    return config
+};
 
-    console.log('Going to app.parkalot.io...')
+const job = cron.schedule('1 1 * * *', (async () => {
```

Comment:
what do you think to have the schedule also read from config?

Reply:
could be done, just want to consider reading config before job is started